CONTRIBUTING.md

```
@@ -0,0 +1,243 @@
+- Medium to large-scale feature additions (e.g., features requiring new Config / new menu additions, etc.)
+- Major UI refreshes
+
+<sub>Branch names will be mentioned later.</sub><br>
+ブランチ名については後ほど後述します。<br>
+
+## Commit Message
+
+<sub>Commit messages in this repository are described as follows. (from 2026/01/10)</sub><br>
+本リポジトリでのコミットメッセージは、下記のように記述します。(2026/01/10~)<br>
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)